recipes/optunahub/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "optunahub" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/optunahub-{{ version }}.tar.gz
+  sha256: 6fc08afcdb9ca35dc1badfa4a8152c7645f3c25f125d011e08ac897f13152d0a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=61.1.0
+    - wheel
+    - pip
+  run:
+    - python >=3.8
+    - ga4mp
+    - optuna
+    - pygithub >=1.59
+
+test:
+  imports:
+    - optunahub
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  summary: Python library to use packages in OptunaHub
+  home: https://github.com/optuna/optunahub
+  license: MIT
+  license_file: LICENSE
+  doc_url: https://hub.optuna.org/
+  dev_url: https://github.com/optuna/optunahub
+
+extra:
+  recipe-maintainers:
+    - y0z
+    - nabenabe0928
+    - c-bata
+    - contramundum53
+    - HideakiImamura
+    - not522
+    - toshihikoyanase
```

Comment:
@y0z @nabenabe0928 @contramundum53 @HideakiImamura @not522 @toshihikoyanase Could you please comment on this pull request if you are willing to be listed here?

Reply:
Sure, I'm happy to be listed. Thanks!